docs/guide/search/elastic.md

```diff
@@ -0,0 +1,573 @@
+# Elasticsearch search engine
+
+!!! enterprise
+
+    Elasticsearch is an open-source, distributed, Java-based search engine that responds to queries
+    in real-time and is easily scalable in reaction to changing processing needs.
+
+    With this document you learn to configure Elastic and eZ Platform to work together.
+    For a detailed description of advanced settings that you might require in a specific
+    production environment, see the documentation provided by Elastic, beginning with the [Set up Elasticsearch](https://www.elastic.co/guide/en/elasticsearch/reference/7.7/setup.html) section.
+
+    !!! tip "**Prerequisite**"
+
+        Before you proceed to configuring the integration between Elasticsearch and eZ Platform, make sure
+      	that you install a standalone instance of Elasticsearch locally and experiment with it
+      	to become familiar with how indexing, filtering and queries work.
+
+    ## How to set up the Elasticsearch search engine
```

Comment:
How about removing this heading and bumping all other headings one level up? That will give a nicer TOC